### ISSUE TITLE
Bug - Unfollow maxout at 246

### ISSUE THREAD

**Issue by jeffreylm** [InstaPy/InstaPy#490]
Hello Everyone,

The script ends the session after it reaches 246 unfollows.
There us no error of any kind. Does anyone know why this is happening?

@BernardoGO It has your fix, but not the latest merge as of yet.

**Comment by jeffreylm** [comment]
@timgrossmann Hello Tom,

No one seems to be looking into this bug.  Could you look into why this is happening?
It now end at times on an even lower number with the last update. It ended at 58, whilst I asked for 200.

Greetings.
Jeffrey

**Comment by timgrossmann** [comment]
@jeffreylm I hope you can understand that this is openSource project and Myself and every one else does this in his free time. I don't get a single dollar for this so I don't want to sound rude, but please remember that you're not free to demand anything unless you pay for it.

Thank you for understandings this.
I will look into this as soon as I can find some time for it.

**Comment by jeffreylm** [comment]
@timgrossmann Tom,

It wasn't a demand. It was more in the vein of trying to bring light to a bug, and not a feature request. I understand the response for a feature request as in that this is opensource, thant tottaly understandable.

This bug it one of the core features of the bot, and sooner or later others will also experience it. I just ran the bot, and not it unfollows 0. So for some reason it gets worse over time. Before the latest update I could garantee get 246 unfollows, now it went from 246 to 40 to now zero. Multiple restart had no effect.
Also **_Number of people followed automatically remaining_**,  isn't going down with each set of unfollow sessions. Just for clarity, the followed nr on instagram is going down, so it was really removing people, just not updating the database to reflect that.

**Comment by jeffreylm** [comment]
@timgrossmann Just for clarity, I do understadn that you are doing this in your free time, and I really appreciate the work you have done.

One thing I just tried was to restart the computer (Win 7, Python 3.6.2), and your bot doesn't unfollow anymore.

session.unfollow_users(amount=400, onlyInstapyFollowed=True)

Greetings,
Jeffrey

**Comment by jeffreylm** [comment]
@timgrossmannann 

When I now change session.unfollow_users(amount=400, onlyInstapyFollowed=True) into
session.unfollow_users(amount=400). It started unfollowing.
Then I stopped it and put the end part back, and it was back to not unfollowing anyone, and ending the session and closing the browser.

Side note, it does scroll down and do all that it normally does to get ready to start unfollowing, then just sits there and sorts of times out and ends the session.

Greetings,
Jeffrey

**Comment by timgrossmann** [comment]
@jeffreylm Maybe you don't have any accounts that were followed by InstaPy left?

**Comment by jeffreylm** [comment]
I ran it for the entire day yesterday, and added about 700 people, then I edited the script to only do an unfollow today.

**Comment by timgrossmann** [comment]
@jeffreylm Well, looks like a bug then

**Comment by jeffreylm** [comment]
Yes, I really wish I had the knowledge to fix it, else I would, but this is outside of what I know at the moement about python. And as I stated above, before the recent update I could always get 246 unfollows, but now none, if onlyInstapyFollowed=True is set.

**Comment by jeffreylm** [comment]
Also this is what the command prompt write during execution : Number of people followed automatically remaining: 2249

It was lower before I did the all day run yesterday. One thing to also note, Instapy has never lowered that number ever since I am running the script, it only adds to the value, even when onlyInstapyFollowed=True was being used.

@timgrossmann 

Greetings,
Jeffrey

**Comment by jeffreylm** [comment]
@timgrossmann 

I opened the followed pool csv file and looked at the bottom of the list, and the account that was asked to follow it's users: session.follow_user_followers(['friend', 'friends', 'friends3'], amount=150, random=True)

Has many follows of the given account entered in the csv file. So for instance if it was elon musk, there is about 150 times his name in the file tight under each other.

**Comment by jeffreylm** [comment]
The followed.txt has "0" at the first line.

**Comment by jeffreylm** [comment]
@BernardoGO 

Hello Bernado,

I am having an issue with the unfollow as mentioned above. Would you be so kind as to look into what and why it is happening? I have tried everything that I could thing of, and having beginners knowledge of python, makes it impossibel for me to figure it out for now.


Side note I updated to the latest version today and re-trired, same issue.

One other piculiar thing is I went ahead yesterday and followed new people again, then did the unfollow only onlyInstapyFollowed and and with it set to 500, it would unfollow 150, then end session, then it reached to a unfollow of 118, then ended session, then unfollowed 14, then ended session, and on the last try using start,bat, it unfollow was zero. 

And just for clarity. I had followed about 700-800 people and my  jeffreylrm_followedPool.csv still has many people in it, and when the script started and ended with zero unfollowed, the propmt read : 

C:\Users\jeffrey\InstaPy>py quickstart.py
Logged in successfully!
Number of people followed automatically remaining: 2242
--> Total people unfollowed : 0

Session ended




Greetings,
Jeffrey

**Comment by jeffreylm** [comment]
@BernardoGO 

The unfollow use to max out at 246, after several updates it now maxes out at 150. where I have to restart the bat file to for it to at it again.

**Comment by jeffreylm** [comment]
@BernardoGO @timgrossmann 
What I have come to realize is that there may not be anyone from the follow pool on the screen that chrome has scrolled to. I tested it by opening the followpool csv window and doing a search of few of the names randomly. And I noticed that none were found from the Chrome window.

So my question is.
Does the script actually set the window up to have follow pool people? So is there a loop that does, if no follow-pool then scroll down. Then that can help to explain why the follow-pool was getting lower with each run, because it was based on chance, but it does not explain the 246 max unfollow per session.

Thank you in advance.
